parts/k8s/containeraddons/kubernetesmasteraddons-omsagent-daemonset.yaml

```diff
@@ -507,7 +486,7 @@ spec:
             - mountPath: /etc/omsagent-secret
               name: omsagent-secret
               readOnly: true
-            - mountPath: /etc/config
+            - mountPath : /etc/config
```

Comment:
Looks like there is an extra space after "mountPath" and this might cause a problem.